.github/workflows/windows.yml

```diff
@@ -313,11 +313,13 @@ jobs:
           cmake --install ./build/ --config ${{ matrix.build-type }} --prefix ${{ env.OV_INSTALL_DIR }}
 
       - name: Build samples (Release)
+        # TODO Update Windows and macOS workflows
         if: ${{ 'Release' == matrix.build-type }} # build_samples enforces Release build
         run: |
           & ${{ env.OV_INSTALL_DIR }}\samples\cpp\build_samples.ps1 -i ${{ github.workspace }}/samples_install
 
       - name: Build samples (Debug)
+        # TODO Update Windows and macOS workflows
```

Comment:
what needs to be updated specifically?

I think after https://github.com/openvinotoolkit/openvino.genai/pull/826 being merged, we will have better picture with debug builds.